util/envDynawo.sh

```diff
@@ -1306,6 +1306,7 @@ generate_preassembled() {
   if ! is_launcher_installed; then
     install_launcher || error_exit "Error during launcher installation."
   fi
+  export OPENMODELICAHOME=$DYNAWO_INSTALL_OPENMODELICA
```

Comment:
export_var_env_force

Reply:
If I use export_var_env_force, I get this error : "You must export variables with DYNAWO prefix for OPENMODELICAHOME."

Reply:
indeed

Reply:
You could also add OPENMODELICA as an exception in the export_var_env_force